CMakeLists.txt

```diff
@@ -361,6 +361,16 @@ function(generateInstallDirectives)
       RENAME "LICENSE.txt"
     )
 
+    install(
+      DIRECTORY "packs"
+      DESTINATION "."
+    )
```

Comment:
I'm a bit torn about this. Mostly, I think the packs are kinda stale, so I don' really want to ship them.

I don't mind being overridden though

Reply:
I agree about staleness and I'm of the same opinion, we could ship them for consistency or not.

Reply:
Sure, I'll need to make a few changes to the chocolatey installer scripts but I think it should be relatively minimal to remove the `packs` bundling alltogether.